submitted_models/bosdyn_spot/urdf/camera.xacro

```diff
@@ -20,7 +20,7 @@
                     <update_rate>30</update_rate>
                     <visualize>${visualize}</visualize>
                     <camera>
-                        <horizontal_fov>${radians(95)}</horizontal_fov>
+                        <horizontal_fov>${radians(102.35)}</horizontal_fov> <!-- computed from real fx -->
```

Comment:
Is the update to the camera FoV based upon information from CoSTAR or something else?

Reply:
Yes, it's computed from the `fx` provided by CoSTAR.